tests/kafkatest/services/templates/kafka.properties

```diff
@@ -20,6 +20,8 @@ port=9092
 #host.name=localhost
 advertised.host.name={{ node.account.hostname }}
 #advertised.port=<port accessible by clients>
+listeners=PLAINTEXT://:9092,SSL://:9093
```

Comment:
I wrote the above before I saw @rajinisivaram's message. @rajinisivaram, one thing to keep in mind is that interbroker_protocol only makes a difference if there are multiple brokers. If the tests are only using a single broker, then switching that property is a bit pointless. In the case where we have multiple brokers, I agree with the test combination you suggested.